README.md

```diff
@@ -1,2 +1,257 @@
+- Make sure that you use `exec=` and not `name=` in `node` tags.
+- Make sure that the launch files you `include` have the right suffix (probably `.launch.xml` for your's, probably `.launch.py` for externals, but not `.launch`: this is probably a ROS1 artifact).
+
+
+### Gazebo, simulation, URDF and navigation migration
+1. Need to pass `use_sim_time:=true` to every node in launch. Use special operations to set a parameter in every node (`set_parameter` tag at global scope in XML).
+2. Use [this page](https://github.com/ros-simulation/gazebo_ros_pkgs/wiki) to check how to migrate a given gazebo-ros plugin.
+3. For you models to appear in Gazebo, you might need a line [like this one in your `package.xml`](https://github.com/introlab/opentera-webrtc-ros/blob/ros2/turtlebot3_beam_description/package.xml#L26).
+4. The differential drive plugin has a `odometry_source` option. It used to be a string, now it's an integer (see [here](https://github.com/introlab/opentera-webrtc-ros/blob/ros2/turtlebot3_beam_description/urdf/turtlebot3_waffle.gazebo.xacro#L71)).
+5. Starting gazebo from a launch file is different. Same goes for the `robot_description`, which was a global parameter in ROS1, and is now a normal parameter to the `robot_state_publisher` node in ROS2 and received via the `/robot_description` topic by any other node (published by `robot_state_publisher`). Compare [before](https://github.com/introlab/opentera-webrtc-ros/blob/ros1/opentera_webrtc_demos/launch/opentera_turtlebot_sim.launch#L27-L42) and [after](https://github.com/introlab/opentera-webrtc-ros/blob/ros2/opentera_webrtc_demos/launch/opentera_turtlebot_sim.launch.xml#L26-L41).
+6. Every reference to `move_base` need to be replaced with the ROS2 `nav2` equivalent. Compare [before](https://github.com/introlab/opentera-webrtc-ros/blob/ros1/opentera_webrtc_demos/launch/opentera_turtlebot_sim.launch#L98-L101) and [after](https://github.com/introlab/opentera-webrtc-ros/blob/ros2/opentera_webrtc_demos/launch/opentera_turtlebot_sim.launch.xml#L99-L101) for a simple example in launch file.
+
+### RVIZ config files migration
+1. The `rviz` package is now `rviz2`. Replace `rviz` with `rviz2` in your launch files.
+2. All the components have moved. They no longer have the `rviz` prefix, but usually `rviz_common` or `rviz_default_plugins`. Use existing RVIZ config files for ROS2 to check the new names.
+3. A bunch of stuff changes in configuration of components
+  1. `robot_description` received via topic
+  2. Different component altogether to send goals to `nav2` than used with ROS1 `move_base`
+  3. A bunch of others.
+
+The easiest way is probably to re-create the config file from scratch by re-adding and re-configuring the components you need.
+Checking a diff between your old config file and a new one using similar components can also work, migrating the changes parts that seem important, but not touching window sizes and stuff like that.
+
+## Other tips
```

Comment:
Add a tip about multi robot configuration and node/topic/service discovery with https://docs.ros.org/en/foxy/Concepts/About-Domain-ID.html ROS_DOMAIN_ID.